_core/smart/tutorial-deploy-call.md

```diff
@@ -0,0 +1,140 @@
+
+![faucettx](images/faucet-transfer.png)
+
+## Step 3: Deploying the counter contract
+
+Go back to the Sandbox screen and switch to the ["Contract deploy"](https://testnet-explorer.blockstack.org/sandbox?tab=contract-deploy) tab. The screen has various options, but we will limit our updates to two fields:
+
+* **Countract name**: Counter
+* **Contract source code**: Contents of the `contracts/counter.clar` file (see [prerequisites](#prerequisites))
+
+![deploy](images/sandbox-deploy.png)
+
+Verify your input and hit "Submit". A confirmation will pop up, indicating that a new contract deploy transaction was issued. Click on "View transaction" to review the pending transaction. You might have to refresh the page a few times. In the next seconds, your contract deployment should be confirmed and you will see a contract creation success screen.
+
+![deploytx](images/contract-deploy.png)
+
+As you can see on the screen, every smart contract's source code is publicly verifiable through the explorer.
+
+## Step 4: Incrementing the counter
+
+To update the counter, we have to call the public `increment` function of the contract. Go back to the Sandbox screen and switch to the ["Contract call"](https://testnet-explorer.blockstack.org/sandbox?tab=contract-call) tab. In the form, enter the following details:
+
+* **Contract address**: Your generated STX address. Hover over the identity component on the right side of the screen to see your full address.
+* **Contract name**: Counter
+
+![sandbox-call](images/sandbox-call.png)
+
+Verify your input and hit "Fetch" to look up available methods. You should see the 3 public methods implemented in the smart contract.
+
+Locate the `(increment)` method, set the post condition mode to "Allow" and hit "Submit". A result confirmation will be displayed, indicating that a new contract call transaction was issued. Click on the transaction to review it. In the next seconds, your contract call should be completed and you will see a contract call success screen. Scroll down to the function summary section to verify your contract call:
+
+![sandbox-call](images/sandbox-calltx.png)
+
+**Congratulations! You just deployed your smart contract and called a public function on the Testnet.**
+
+## Step 5: Get familiar with CLI (optional)
+
+The steps above represent an easy way to get started with contract deployment and contract calls. If you want to stay in the terminal and get access to more advanced capabilities, you should use the Blockstack CLI.
+
+The following set of commands will achieve the same goals as steps 1-4.
+
+```shell
+# Install early release of the new Blockstack CLI for Stacks 2.0
+npm install -g https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx
+
+# Create a new STX address and save keychain details to a file
+blockstack make_keychain -t > new_keychain.txt
```

Comment:
Good idea to save the keychain for later reference. 👍

Reply:
Maybe we should add a comment about why the `-t` flag is used.

Reply:
Also, by saving the output of `make_keychain` to a text file it's no longer visible in the terminal. Perhaps we need instructions to open the file to retrieve the private key.

Reply:
Added notes about how to see STX address and why the `-t` flag is used 👍